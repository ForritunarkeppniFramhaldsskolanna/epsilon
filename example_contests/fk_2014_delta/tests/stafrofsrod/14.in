20
pqvhoi
ncumpgjbayauunk
rab
kgklnq
hzejn
wfudlxlpmzoxxuiz
weluvmnkgsaw
jfdapxartbioystmpfke
ty
ma
zvpkuxlyfg
hzejbjh
wdyskopuqqkogkodvgi
jj
nmncrqwjvdo
yvnsszprhsyvy
sckukyzyre
okbphdycj
zzaihtfbaj
pkthmleopwwdimokuf
