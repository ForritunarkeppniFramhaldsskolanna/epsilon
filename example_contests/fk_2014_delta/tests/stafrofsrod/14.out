hzejbjh
hzejn
jfdapxartbioystmpfke
jj
kgklnq
ma
ncumpgjbayauunk
nmncrqwjvdo
okbphdycj
pkthmleopwwdimokuf
pqvhoi
rab
sckukyzyre
ty
wdyskopuqqkogkodvgi
weluvmnkgsaw
wfudlxlpmzoxxuiz
yvnsszprhsyvy
zvpkuxlyfg
zzaihtfbaj
